packages/teams-js/src/public/video.ts

```diff
@@ -76,23 +167,42 @@ export namespace video {
     /**
      * Current video effect changed
      */
-    EffectChanged,
+    EffectChanged = 'EffectChanged',
     /**
      * Disable the video effect
      */
-    EffectDisabled,
+    EffectDisabled = 'EffectDisabled',
   }
 
   /**
    * Video frame call back function definition
    * @beta
    */
-  export type VideoFrameCallback = (
-    frame: VideoFrame,
+  export type SharedFrameCallback = (
+    frame: VideoFrameData,
     notifyVideoFrameProcessed: notifyVideoFrameProcessedFunctionType,
     notifyError: notifyErrorFunctionType,
   ) => void;
 
+  /**
+   * @beta
+   * Callbacks and configuration supplied to the host to process the video frames.
+   */
+  export type VideoFrameCallbackOptions = {
```

Comment:
These types of objects are typically called "Parameters" in the SDK. Take a look at FilePreviewParameters, AuthenticateParameters, ShowNotificationParameters, AuthTokenRequestParameters, etc. I think a better name for this would be "VideoFrameParameters" or "RegisterForVideoFrameParameters".

Reply:
renamed